clients/search-component/src/TrieveModal/Chat/SuggestedQuestions.tsx

```diff
@@ -23,6 +23,14 @@ export const SuggestedQuestions = ({
     return null;
   }
 
+  const handleSuggestedQuestion = async (q: string) => {
```

Comment:
This needs to be a `useCallback` so it doesn't trigger re-renders inappropriately